src/snowcli/cli/sql.py

```diff
@@ -101,25 +102,25 @@ def execute_sql(
     sys_input = None
 
     if query and file:
-        raise ValueError("Both query and file provided, please specify only one.")
+        raise UsageError("Both query and file provided, please specify only one.")
 
     if not sys.stdin.isatty():
         sys_input = sys.stdin.read()
 
     if sys_input and (query or file):
-        raise ValueError(
+        raise UsageError(
             "Can't use stdin input together with query or filename option."
         )
 
     if not query and not file and not sys_input:
-        raise ValueError("Provide either query or filename argument")
+        raise UsageError("Provide either query or filename argument")
     elif sys_input:
         sql = sys_input
     else:
         sql = query if query else file.read_text()  # type: ignore
 
     if not config.is_auth():
-        raise ValueError("Not authorize")
+        raise UsageError("Not authenticated")
```

Comment:
@sfc-gh-turbaszek I've changed the error message. Can you confirm that I'm right here (that this is about authentication, not about authorization)?

Reply:
+1, I think this is correct